kentikapi/internal/testutil/http.go

```diff
@@ -34,22 +35,26 @@ func NewSpyHTTPHandler(t testing.TB, responseCode int, responseBody []byte) *Spy
 }
 
 func (h *SpyHTTPHandler) ServeHTTP(rw http.ResponseWriter, r *http.Request) {
-	h.RequestsCount++
-	h.LastMethod = r.Method
-	h.LastURL = r.URL
-	h.LastHeader = r.Header
-
-	body, err := ioutil.ReadAll(r.Body)
-	assert.NoError(h.t, err)
-	h.LastRequestBody = string(body)
-
-	err = r.Body.Close()
-	assert.NoError(h.t, err)
-
-	rw.Header().Set("Content-Type", "application/json")
-	rw.WriteHeader(h.responseCode)
-	_, err = rw.Write(h.responseBody)
-	assert.NoError(h.t, err)
+	if r.Method == "PRI" {
+		log.Printf("Omitting HTTP/2 connection preface (PRI)\n")
+	} else {
```

Comment:
Please just call return in `if` block instead of adding `else` block.
The same for MultipleResponseSpyHTTPHandler

Reply:
Done